.github/workflows/slack.yaml

```
@@ -0,0 +1,36 @@
+# Send a message to development when the docker image changes!
+# Firstly made by Finn Stevenson!
+
+name: Docker Slack Message
+
+# Control when this will run!
+on:
+  #testing on pull request, sends a message everytime
+  # pull_request:
+  #   branches: [main]
+  #   paths:
+  #     - 'docker/**'
+
+  push:
+    branches:
+      - main
+    paths:
+      - 'docker/**'
+
+
+
+jobs:
+  slack-notifcations:
+    runs-on: ubuntu-20.04
+    name: Send a message to slack when the docker folder has changes
```

I would amend this slightly, because the intent is to notify when the docker build files/scripts change - which is basically the same thing in practice. However, I think it makes it clearer why we won't notify when the robocup image build files change, for instance. I might be overthinking this though. What do you think @Y0sh1dk 

```suggestion
    name: Send a message to slack when the main docker image's build files change
```

---

Yeah I think thats fair enough, as we will probably implement a similar slack notification for the robocup image once once #745 is complete.